tests/OracleTest.php

```diff
@@ -34,9 +34,11 @@ public function testDetection()
 
     public function connect($ver = '')
     {
+        $version = $ver ? "\\Version{$ver}" : '';
+
         return new \atk4\dsql\Connection(array_merge([
             'connection' => new \PDO('sqlite::memory:'),
-            'query_class' => \atk4\dsql\Query_Oracle::class . $ver,
+            'query_class' => "\\atk4\\dsql\\Oracle{$version}\\Query",
```

Comment:
you use single quotes string here...

for prod/non test I qould use probably something like `['' => ::class, ...][$ver]`...

Reply:
What do you mean single quotes string?

Reply:
`'\atk4\dsql\Oracle' . $version . '...` should be the preffered format

- we should use single quoted strings everywhere
- do not escape `\` when not needed